cmd_clean/main.go

```diff
@@ -88,11 +88,17 @@ func main() {
 	// customer http client should handle verify and authentication
 	// here using the basic user/password mode for authentication
 	// not handling verify
+	// the error roughly are:
+	// 1. failed to validate config
+	// 2. failed to get nsx client
+	// 3. failed to initialize cleanup service
+	// 4. failed to clean up specific resourc
 	var err error
 	var status clean.Status
 	if useExternalHttp {
 		tr := &http.Transport{
 			IdleConnTimeout: 30 * time.Second,
+			// #nosec G402: ignore insecure options
```

Comment:
what is this comment for?

Reply:
this is for golangci-lint tool skip the check for this,